client/src/components/MyCalendar.jsx

```diff
@@ -116,50 +111,124 @@ class MyCalendar extends React.Component {
-            }</button>
-          <button onClick={this.showCalendar.bind(this, true, 'out')}>
-            {!this.state.endDate
-              ? this.state.checkOut
-              : this.state.endDate.toLocaleDateString()
-            }</button>
-
-          {this.state.showCalendar === true
-            ?
-            <Calendar
-              returnValue={'range'}
-              calendarType={'US'}
-              selectRange={true}
-              onClickDay={this.handleClickDay}
-              prev2ButtonDisabled={true}
-              showNeighboringMonth={false}
-              onChange={this.onChange}
-              tileDisabled={({date}) => {
-                return !this.state.activeDays.map((date) => date.toLocaleDateString())
-                  .includes(date.toLocaleDateString());
-              }}
-              activeStartDate={!this.state.startDate ? new Date() : this.state.startDate}
-              value={[
-                (!this.state.startDate ? new Date() : this.state.startDate),
-                (!this.state.endDate ? null : this.state.endDate)
-              ]}/>
-            :
-            <div>Calendar hidden</div>
+      <div className='calendar-container'>
+
+        <ClickOutHandler onClickOut={this.showCalendar.bind(this, false)}>  
+
+          <div className='calendar-button-holder selector-box'>
+            <button className='calendar-btn' onClick={this.handleCheckInClick.bind(this)}>
+              {
+                !this.state.startDate 
+                  ? this.state.checkInDisplayText 
+                  : this.state.startDate.toLocaleDateString()
+              }
+
+            </button>
+            <button className='calendar-btn' onClick={this.handleCheckOutClick.bind(this)}>
+              {
+                !this.state.endDate 
+                  ? this.state.checkOutDisplayText 
+                  : this.state.endDate.toLocaleDateString()
+              }
+            </button>
+          </div>
+
+          {
+            <div id='myModal' className='modal'>
+
+              <Calendar 
+                returnValue={'range'}
+                calendarType={'US'}
+                selectRange={true}
+                onClickDay={this.handleClickDay}
+                prev2ButtonDisabled={true}
+                showNeighboringMonth={false}
+                onChange={this.onChange}
+                formatS
+                tileDisabled={({date}) => {
+                if (!this.state.activeDays) return true;
+                  return !this.state.activeDays.map((date) => date.toLocaleDateString())
+                    .includes(date.toLocaleDateString());
+                }}
+                value={
+                  [this.state.startDate || this.state.endDate || new Date(), 
+                    this.state.endDate || this.state.startDate]
+
+                }/> 
+              <div className='caption'>
+                Minimum stay varies
+              </div >
```

Comment:
This is marked red, not sure why or how, but maybe because there is an extra space?